docs/000-intro.apib

```diff
@@ -111,3 +111,23 @@ The common [HTTP Response Status Codes](http://httpstatus.es) are used. You
 know the drill.
 
 For 400-family status codes, the response body will contain an array of errors.
+
+
+## Group Pagination
+
+In order to keep our servers happy, the Packet API will automatically paginate the requested items for all resources and collections.
```

Comment:
Instead of saying "keep our servers happy", which implies that our servers are slow, I'd say "keep responses fast".